### PR TITLE
[QA] Add trigger for component library tests

### PR DESCRIPTION
**Pull request by l0uden**
## Description

Added component library tests trigger
Connected with https://github.com/mckinsey/vizro-qa/pull/119

## Notice

- [x] I acknowledge and agree that, by checking this box and clicking "Submit Pull Request":

  - I submit this contribution under the [Apache 2.0 license](https://www.apache.org/licenses/LICENSE-2.0.txt) and represent that I am entitled to do so on behalf of myself, my employer, or relevant third parties, as applicable.
  - I certify that (a) this contribution is my original creation and / or (b) to the extent it is not my original creation, I am authorized to submit this contribution on behalf of the original creator(s) or their licensees.
  - I certify that the use of this contribution as authorized by the Apache 2.0 license does not violate the intellectual property rights of anyone else.
  - I have not referenced individuals, products or companies in any commits, directly or indirectly.
  - I have not added data or restricted code in any commits, directly or indirectly.

### CODE REVIEW

File: .github/workflows/test-integration-vizro-core.yml [hunk 1]
```diff
@@ -41,27 +41,6 @@ jobs:
     steps:
       - uses: actions/checkout@v4
 
-      # Pinned Chrome version because chromedriver v120 recognized as chrome-headless-shell instead of chrome
-      # https://github.com/plotly/dash/issues/2712
-      # https://bugs.chromium.org/p/chromedriver/issues/detail?id=4665
-      # https://github.com/mckinsey/vizro/pull/215/
-      # If these are fixed then this whole step can be removed.
```
Comment (huong-li-nguyen) [inline .github/workflows/test-integration-vizro-core.yml:48]: I guess this got fixed now? 🥳

File: .github/workflows/vizro-qa-tests-trigger.yml [hunk 2]
```diff
@@ -50,6 +51,8 @@ jobs:
             export INPUT_WORKFLOW_FILE_NAME=${{ secrets.VIZRO_QA_NOTEBOOKS_TESTS_WORKFLOW }}
           elif [ "${{ matrix.label }}" == "vizro-ai ui tests" ]; then
             export INPUT_WORKFLOW_FILE_NAME=${{ secrets.VIZRO_QA_VIZRO_AI_UI_TESTS_WORKFLOW }}
+          elif [ "${{ matrix.label }}" == "component library tests" ]; then
+            export INPUT_WORKFLOW_FILE_NAME=${{ secrets.VIZRO_QA_VIZRO_COMPONENT_LIBRARY_TESTS_WORKFLOW }}
```
Comment (huong-li-nguyen) [inline .github/workflows/vizro-qa-tests-trigger.yml:55]: What is this component library tests? Are these the KPI cards used in a pure dash app?

**Comment by huong-li-nguyen** [comment]
@l0uden - would you mind adding some description of what these changes are for? 😄 I have an assumption, but would be good to know for future tracking!

**Comment by l0uden** [comment]
> @l0uden - would you mind adding some description of what these changes are for? 😄 I have an assumption, but would be good to know for future tracking!

Sorry, I've definitely added the description, but looks like it wasn't saved.

File: .github/workflows/vizro-qa-tests-trigger.yml (hunk 2 above)
Comment (l0uden) [reply .github/workflows/vizro-qa-tests-trigger.yml:55]: Yes, and now I've added link to the vizro-qa PR with the tests

File: .github/workflows/test-integration-vizro-core.yml (hunk 1 above)
Comment (l0uden) [reply .github/workflows/test-integration-vizro-core.yml:48]: Yes!

**Review by huong-li-nguyen — APPROVED** [review]

File: .github/workflows/test-integration-vizro-core.yml (hunk 1 above)
Comment (antonymilne) [reply .github/workflows/test-integration-vizro-core.yml:48]: Awesome that we can get rid of this 🎉

**Review by antonymilne — APPROVED** [review]

**Pull request merged**